### DISCUSSION TITLE
Tracking external issues for bug fixing and improvements

### DISCUSSION THREAD

**Discussion by edoardocavazza** [chialab/rna#16] [General]
### 🐛 Bugs

* Invalid coverage (https://github.com/evanw/esbuild/issues/1448)
* Duplicated import statements for the same source (https://github.com/evanw/esbuild/issues/475)

### ⚡️ Speed

* Core URL assets (https://github.com/evanw/esbuild/issues/795)
* ES5 support (https://github.com/evanw/esbuild/issues/297, https://github.com/evanw/esbuild/issues/192)

### 🧰 Maintenance

* Webpack comments (https://github.com/evanw/esbuild/issues/1240)
* JSX import source (https://github.com/evanw/esbuild/issues/1172)
* Improve HTML generation (https://github.com/evanw/esbuild/issues/31)
* Use mocha spec reporter (https://github.com/modernweb-dev/web/issues/229)

**Comment by okikio** [comment]
Quick question, I'd like to use `@chialab/esbuild-plugin-worker` together with `gulp-esbuild` as a plugin for loading workers, but I am not clear on how I'm supposed to do that. the docs don't mention how you create workers.
